### DISCUSSION TITLE
How to install ImageMagick use it with pdfplumber (in a conda environment)

### DISCUSSION THREAD

**Discussion by falloutats** [ImageMagick/ImageMagick#6290] [Help] [answered]
### ImageMagick version

latest

### Operating system, version and so on

macOS Monterey (version 12.6.5)

### Description

Required specific steps to install and successfully use both the libraries. I have tried using 'brew' but that has not worked. Please let me know If you need more information.

**Comment by gaojunran** [comment, marked as answer]
Just `conda install conda-forge::pdfplumber`, and it will resolve imagemagick as its dependencies.

**Reply by falloutats** [reply]
that was quick : )
